packages/datadog-plugin-jest/src/index.js

```diff
@@ -60,6 +83,14 @@ function createHandleTestEvent (tracer, testEnvironmentMetadata) {
       [SAMPLING_PRIORITY]: AUTO_KEEP,
       ...testEnvironmentMetadata
     }
+    let testParameters = params[event.test.name]
+    if (testParameters && Array.isArray(testParameters)) {
+      try {
+        testParameters = testParameters.shift()
+        commonSpanTags[TEST_PARAMETERS] = JSON.stringify(testParameters)
+        // eslint-disable-next-line
+      } catch (e) {}
```

Comment:
It would be more explicit to add a comment in the catch block with a mention of why it's fine to swallow the error instead of ignoring eslint.

Reply:
done in https://github.com/DataDog/dd-trace-js/pull/1345/commits/36882794d13d37e55619b5a38113532976cbf66f